src/util/string.h

```diff
@@ -84,6 +84,14 @@ inline QString convertWCStringToQString(
     return QString::fromWCharArray(wcs, static_cast<int>(wcsnlen_s(wcs, maxLen)));
 }
 
+inline QString removeTrailingWhitespaces(const QString& str) {
+    QString out = str;
+    while (out.endsWith(' ')) {
```

Comment:
Maybe it would make sense to handle other kinds of whitespace that one may encounter when pasting something from somewhere else? For example, non-breakable spaces.

Although, after checking the sources and against my expectations, [`QString::trimmed()`](https://github.com/qt/qtbase/blob/55b5e78ac31171be61ceeb932d494d5060d1fa02/src/corelib/text/qstringalgorithms_p.h#L67-L72) also [only](https://github.com/qt/qtbase/blob/55b5e78ac31171be61ceeb932d494d5060d1fa02/src/corelib/text/qstringalgorithms_p.h#L30) [handles](https://github.com/qt/qtbase/blob/96c07655c46afdd3e2da7ef7d339b518d36c83c5/src/corelib/text/qlocale_p.h#L573-L577) the [ASCII whitespace characters](https://github.com/qt/qtbase/blob/96c07655c46afdd3e2da7ef7d339b518d36c83c5/src/corelib/text/qlocale_p.h#L70-L76), and none of the UTF-8/Unicode Whitespace Characters. This is opposed to e.g. .NET [`String::Trim()`](https://github.com/dotnet/runtime/blob/5535e31a712343a63f5d7d796cd874e563e5ac14/src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs#L2140-L2147) which [supports all Unicode whitespace characters](https://learn.microsoft.com/de-de/dotnet/api/system.char.iswhitespace?).